changelog/0048-update-supervisor-confs.yml

```diff
@@ -0,0 +1,21 @@
+title: update-supervisor-confs
+key: update-supervisor-confs
+date: 2022-05-25
+optional_per_env: no
+min_commcare_version: 0b1ff6e1111f3280b56fa66ff2fd94bd8d46e55c
+max_commcare_version:
+context: |
+  Run management command to update shell runner scripts for django and celery.
```

Comment:
I associate the phrase "management commands" with Django `manage.py` commands (which in the context of commcare-cloud are invoked with `cchq <env> django-manage ...`) and would have phrased this as "Run command to update ..."

Not a big deal.

Reply:
Ahh, you're right @millerdev. Do you think it's worthwhile updating? Shouldn't be hard.